src/Widgets/UserListBox.vala

```diff
@@ -36,20 +36,38 @@ namespace SwitchboardPlugUserAccounts.Widgets {
 
             //only build the guest session list entry / row when lightDM is X11's display manager
             if (get_display_manager () == "lightdm") {
-                var avatar = new Granite.Widgets.Avatar.with_default_icon (32);
+                var avatar = new Hdy.Avatar (32, null, false);
+
+                // We want to use the user's accent, not a random color
```

Comment:
This is haven't been implemented yet, right?

Reply:
For the guest account yes, removing the color classes allows it to fall back to the current user's accent color